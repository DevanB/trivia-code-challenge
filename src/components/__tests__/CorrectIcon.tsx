import {render} from '@testing-library/react'
import * as React from 'react'
import {CorrectIcon} from '../CorrectIcon'

describe('<CorrectIcon/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<CorrectIcon />)
    expect(container).toMatchSnapshot()
  })
})
