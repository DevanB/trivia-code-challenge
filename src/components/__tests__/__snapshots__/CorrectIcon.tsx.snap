// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CorrectIcon/> should render without crashing 1`] = `
<div>
  <div
    class="CorrectIcon__Icon-ykms1h-0 fhYAOa"
  >
    <svg
      viewBox="0 0 20 20"
    >
      <path
        d="M0 11l2-2 5 5L18 3l2 2L7 18z"
        fill="currentColor"
      />
    </svg>
  </div>
</div>
`;
